test/session/with-jwt/jwt.override.test.js

```diff
@@ -0,0 +1,208 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Why have you named these files with `.override.test`? Rename them to `jwtOverride.test.js`? Or maybe this is fine.. not sure.

Reply:
No particular reason, it can be called either. I can change it to `jwtOverride.test.js` if thats more preferable

Reply:
What you have now is fine..